src/services/NotificationService.ts

```diff
@@ -21,30 +18,23 @@ class NotificationService {
    * Sending email with the certificate according to the given params
    * @param params - personalization details,email and certificate
    * @param emails - emails to send to
-   * @param emailType - email receiver type
+   * @param activityId - activityId for logging purposes
    */
-  public async sendNotification(params: any, emails: string[], emailType: string, activityId: string): Promise<any[]> {
+  public async sendNotification(params: any, emails: string[], activityId: string) {
     const templateId: string = await this.config.getTemplateIdFromEV();
     const emailDetails = {
       personalisation: params,
     };
-    const sendEmailPromise = [];
 
     for (const email of emails) {
-      const sendEmail = this.notifyClient.sendEmail(templateId, email, emailDetails).then((response: any) => response.data);
-      sendEmailPromise.push(sendEmail);
+      try {
+        await this.notifyClient.sendEmail(templateId, email, emailDetails);
+        console.log(`report successfully sent email to ${email} for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
+      } catch (error) {
+        console.log(`failed to send for ${JSON.stringify(email)}`);
```

Comment:
same concern here. Do we definitely want to log these adresses as it will include the ATF contact and the individual tester. @m-coslett do we know if those are both ok from a GDPR perspective to end up in the logs?

I think when we talked about this issue in regards to tokens, we decided it was fine...

Reply:
imo I can't see it being an issue, the things we have are surely publically accessible too